### ISSUE TITLE
Converting other backbones

### ISSUE THREAD

**Issue by cancam** [cccorn/AP-loss#7]
Hi,

I am planning to use different backbones (resnext-101, resnet-152) different than that you've already provided. As you said in readme, the backbone models are converted from caffe version. Could you please provide the converter tool that you used?

Thank you for your time.
Baris.

**Comment by cccorn** [comment]
Our converter code only works for ResNet-50 and ResNet-101. For ResNet-152 and ResNeXt-101, I suggest using the model provided by Pytorch: 
[https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py](url)
and then change the backbone description accordingly in 

> lib/model/model.py 

and image normalization method in 

> lib/dataloader/augmentation.py, SubtractMeans() 

for training, and in 
> lib/dataloader/dataloader.py, Normalizer() 

for testing.

**Comment by suresh-s** [comment]
I also like to change to the backbone to resnet152.  why changes required in image normalization method (`lib/dataloader/augmentation.py, SubtractMeans()`) and in `lib/dataloader/dataloader.py, Normalizer()`.  Could you tell me?

and i want to check detection in test images using the trained model, is there any demo code in this repository?
